.github/workflows/go.yml

```diff
@@ -66,6 +66,10 @@ jobs:
           env_vars: GO_VERSION,TEST_MONGODB_IMAGE
           fail_ci_if_error: false
 
+      - name: Dry-run GoReleaser
+        run: |
+          make release-dry-run
+
```

Comment:
it will run on every matrix element which isn't great, let's make another job and rename current one to test instead of build

Reply:
yeah reworked them now.